<?xml version="1.0"?> 
<launch>

    <node pkg="cmd_vel_prioritizer" type="cmd_vel_prioritizer" 
		respawn="true" name="cmd_vel_prioritizer" output="screen">

        <param name="output_topic" value="turtle1/cmd_vel"/>
        
        <param name="input_topic1" value="teleop/cmd_vel"/>
        <param name="input1_locktime" value="4.0"/>
        
        <param name="input_topic2" value="none"/>
        <param name="input2_locktime" value="0.2"/>
        
        <param name="input_topic3" value="none"/>
        <param name="input3_locktime" value="0.2"/>
        
        <param name="input_topic4" value="move_base/cmd_vel"/>
        <param name="input4_locktime" value="0.2"/>
        
        <param name="pub_frequency" value="10.0"/>
  
    </node>

</launch>
